guides/source/upgrading_ruby_on_rails.md

```diff
@@ -530,6 +530,11 @@ To:
 Rails.application.secrets[:smtp_settings][:address]
 ```
 
+### `render :text` Option is Removed
```

Comment:
As per the [release notes](https://github.com/rails/rails/blob/master/guides/source/5_1_release_notes.md#removals-1), `:nothing` _and_ `:text` were deprecated.

Reply:
good call, thanks!